frontend/src/components/elements/ExceptionElement/styled-components.ts

```diff
@@ -0,0 +1,17 @@
+import styled from "@emotion/styled"
+
+export const StyledStackTraceRow = styled.div(({ theme }) => ({
+  marginTop: theme.spacing.sm,
+  "&:first-of-type": {
```

Comment:
So Yes. I did make the change. Base on how we use it, it doesn't change anything.

I changed it because Emotion throws a warning for `:first-child`. TLDR it may not work for Server Side Rendering. It doesn't really matter to us I believe, but I figure if both work, let's do the one Emotion is happier with.

https://stackoverflow.com/questions/24657555/what-is-the-difference-between-first-child-and-first-of-type does a good explanation of the difference.

Reply:
Wouldn't this then need to be `*:first-of-type` if you want to replace `:first-child`? Or perhaps I'm misunderstanding how Emotion translates things